docker/README.md

```diff
@@ -0,0 +1,77 @@
+# README
+This project provide three different docker images.
+* Arthur
+* Arthur Server
+* Arthur Worker
+
+## ARTHUR
+This docker image run a standalone Arthur service (server and worker).
```

Comment:
Hmm, I have understood it once I read the below comments. So Artthur is the command docker image in which is based the Arthur Server docker image (which runs arthur service) and the Arthur Worker docker image (which runs the worker service). Ok!